### DISCUSSION TITLE
How to get New Ui with docker

### DISCUSSION THREAD

**Discussion by whizkraft948** [blockscout/blockscout#8370] [Q&A] [answered]
i am trying to learn how to set up blockscout using docker, but it will always shows me the older version ui. Not the newly launched ui.
Can anyone help please

**Comment by ulyanas** [comment, marked as answer]
@whizkraft948  Please check this guide: https://docs.blockscout.com/for-developers/frontend-migration 
and also Readme here: https://github.com/blockscout/frontend
